packages/utils/modals/DeleteModal.tsx

```diff
@@ -102,7 +104,7 @@ const DeleteModal: React.FC<CommonModalProps<DeleteModalExtraProps>> = ({
         // If we are currently on the deleted resource's page, redirect to the resource list page
         const re = new RegExp(`/${resource.metadata.name}(/|$)`);
         if (re.test(window.location.pathname)) {
-          history.replaceState('/', '');
+          history.goBack();
```

Comment:
what will happen when we delete from list page then ??
we don't want to `goBack` then... how is it working in odf-console ??

Reply:
had the same code in ODF but it is behaving somewhat different

Reply:
this is also important...

Reply:
parked it for now, created https://issues.redhat.com/browse/DFMS-123